src/emulator/storage/crc.ts

```diff
@@ -43,12 +45,14 @@ export function crc32c(bytes: Buffer): number {
 
 /**
  * Adapted from:
- *  - https://github.com/googleapis/nodejs-storage/blob/0c1fa3934a52a608366a8c6c798c43516dd03dbf/src/file.ts#L1406-L1409
+ *  - https://github.com/googleapis/nodejs-storage/blob/1d7d075b82fd24ea3c214bd304cefe4ba5d8be5c/src/crc32c.ts
  */
 export function crc32cToString(crc32cValue: number | string): string {
-  if (typeof crc32cValue === "string") {
-    crc32cValue = Number(crc32cValue);
-  }
-  // Does the reverse of https://stackoverflow.com/q/25096737/849645
-  return "----" + Buffer.from([crc32cValue]).toString("base64");
+  const value = typeof crc32cValue === "string" ? Number.parseInt(crc32cValue) : crc32cValue;
+
+  // Allocating 4 bytes to write an unsigned 32-bit integer
```

Comment:
Would be helpful to link to something that explains the 4 bytes

Reply:
Sg, updated the documentation